jfrog_platform_self_hosted/pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
```

Comment:
To reviewer: This was copied from the [removed pyproject file in the 1.3.0 PR](https://github.com/DataDog/integrations-extras/pull/1895/files#diff-05622f87fe7cf9968249ced34fae97a6de5f74038aecf7f61244bb20889f5b27) but just replacing references to `JFrog Platform` with `JFrog Platform Self Hosted`.